### DISCUSSION TITLE
7 most of the times read as 2

### DISCUSSION THREAD

**Discussion by elcriuz** [jomjol/AI-on-the-edge-device#640] [Q&A]
Hey, any clue how I can fix that it reads 7 as 2 most of the time? Thanks so much for this great tool!
<img width="782" alt="image" src="https://user-images.githubusercontent.com/98161879/152495766-ff189846-093f-4209-acd0-dd0bd9dde59d.png">

**Comment by ckoeber83** [comment]
Hi elcriuz,

Try to get the camera even further to the edge, so that just the numbers which should be read are still visible at the edges. (Much fiddling)

**Reply by elcriuz** [reply]
So you mean I should offset the region of interest from the center of the frame? Like put everything in the lower left edge?

**Reply by ckoeber83** [reply]
I mean closer to the numbers in general (zoom in), that the numbers are bigger on picture.
Here my example:

![grafik](https://user-images.githubusercontent.com/52466072/152509985-a1421e31-1dcb-47f8-8d51-50ea928398cf.png)

**Comment by jomjol** [comment]
Sorry, but two topics:
1) Image quality is really critical 
2) with the first line of the "3" at the bottom it is relatively close to a "2".

The only solution is in one of the next version, which will also be capable of getting "in between" numbers. But this will take some time.

**Reply by elcriuz** [reply]
Thanks for your reply. I am using an ESPCAM32. Any way of improving image quality?

**Comment by jomjol** [comment]
Get closer to the numbers - then they will get bigger and better. You must adjust the focus then.

**Comment by ckoeber83** [comment]
Also do the "CheckDigitIncreaseConsistency = false", this works better with purely digital evaluation.
